CHANGES.md

```diff
@@ -58,7 +58,12 @@
 * FfmpegProcessor, KakaduNativeProcessor, OpenJpegProcessor, and
   PdfBoxProcessor use libjpeg-turbo to write JPEGs if it is available.
 * Added the `scaled` position for image overlays.
-* Updated Apache PDFBox to version 2.0.21.
+* Updated Apache PDFBox to version 2.0.22.
+* PdfBoxProcessor properties added to better control memory
+    * `processor.pdf.scratch_file_enabled` allows use of a scratch file
+    * `processor.pdf.max_memory_bytes` allows control over how much memory a PDF thread consumes (-1 means unlimited)
+    * `processor.pdf.max_storage_bytes` allows control over how much total bytes are used to render a PDF image (memory + temp file)
```

Comment:
In the interest of UX, is there any reason that this key couldn't be removed and max used by PDFBox set to unlimited?

Reply:
My only concern is not having some sort of failsafe on the amount of disk storage allocated to PDFBox... that being said, these are just temporary files that end up getting deleted as soon as the image is rendered and it wouldn't even be a failsafe in the case that many requests came in for a huge file (or requests for multiple pages from the same PDF via Mirador or Internet Archive book viewer).  In those cases there would still be temp files created for each image which could still go over the disk allocation.

Seems safe to remove and use -1 instead by default.